### PR TITLE
Align CI and pipeline setup with behave-html-formatter

### LINKED ISSUE #4: Tests for Python 3.6 are failing
The tests for Python 3.6 are failing. This is due to the setup of the CI runner on GitHub's Azure infrastructure. Python 3.6 is only available on older Ubuntu runners:

```
Version 3.6 was not found in the local cache
Error: Version 3.6 with arch x64 not found
The list of all available versions can be found here:
https://raw.githubusercontent.com/actions/python-versions/main/versions-manifest.json
```

I would suggest to drop version 3.6, unless it's important to you to support older (unsupported) Python versions. I'd also add Python 3.10 and 3.11 to the list of Pythons to run tests against.

I'll start with adapting the Tox and GHA pipeline setup for [behave-html-formatter](https://github.com/behave-contrib/behave-html-formatter/blob/main/tox.ini#L8). Do you want try to fix that yourself for this repository?

### PR DESCRIPTION
**Pull request by bittner**
Updates the CI and pipeline setup, supporting more current Python versions and sporting more linters.

See also: https://github.com/behave-contrib/behave-html-formatter/pull/34
Closes: #4

I've also take the freedom to reformat the code with isort and Black, to make sure the linting jobs all pass. Keeping the minify job identical made it necessary to also align the file names of the CSS theme and JS script files.

### CODE REVIEW

**Comment by bittner** [comment]
All test and linting jobs pass, except for Pylint, which complains for the right, I think.

Please decide for yourselves, whether to split up the module into separate files (I would suggest so) or ignore the warnings via a directive in `pyproject.toml`.

File: tox.ini [hunk 1]
```diff
@@ -80,10 +91,7 @@ html-pretty = behave_html_pretty_formatter:PrettyHTMLFormatter
 
 [flake8]
 exclude = .tox,build,dist,behave_html_pretty_formatter.egg-info
+# see https://github.com/PyCQA/pycodestyle/issues/373
 extend-ignore = E203
 per-file-ignores = ./behave_html_pretty_formatter/__init__.py:F401
-max-line-length = 100
-
-[pylint.main]
-disable = missing-module-docstring,too-many-lines,too-many-instance-attributes,too-many-arguments
```
Comment (fpokryvk) [inline tox.ini:88]: We have changed which warnings we want to disable. Maybe we will split it into multiple files later on.

**Review by fpokryvk — COMMENTED** [review]
Thank you, I will change the pylint configuration to make the tests pass for now.

File: tox.ini (hunk 1 above)
Comment (modehnal) [reply tox.ini:88]: Yeah we can spit it later, no need to rush it for now.

**Review by modehnal — DISMISSED** [review]
Looks good to me, thank you.

**Review by fpokryvk — APPROVED** [review]
Looks good, great work. Thank you all.

File: README.md [hunk 2]
```diff
@@ -6,7 +6,7 @@
 ## Installation
 
 ```shell
-sudo python3 -m pip install behave-html-pretty-formatter
+python3 -m pip install behave-html-pretty-formatter
```
Comment (bittner) [inline README.md:9]: :+1:

**Pull request merged**